scripts/service_migration.py

```diff
@@ -0,0 +1,89 @@
+import copy
+from kubernetes import client, config
+from kubernetes.client.exceptions import ApiException
+import argparse
+import os.path
+from pprint import pprint
+
+IP_ANNOTATION = 'load-balancer-ips'
+def update_annot(v1, svc):    
+    '''
+    ************************************************************************
+    Take dictionary of original service, copies Service.status.load_balancer.ingress[0].ip to Service.metadata.annotations and patches Service
+    --------------
+    Arguments:
+    v1 :: CoreV1Api object
+    svc :: original service as dictionary
+    --------------
+    Returns:
+    None
+    ************************************************************************
+    '''
+    
+    svc['metadata']['annotations'][IP_ANNOTATION] = svc['status']['load_balancer']['ingress'][0]['ip']
+    service = client.V1Service()
+    service.api_version = "v1"
+    service.kind = "Service"
+    service.metadata = svc['metadata']
+    service.spec = svc['spec']
+    service.status = svc['status']
+    try:
+        api_response = v1.patch_namespaced_service(name = svc["metadata"]["name"], namespace="avi-system", body=service)
+        #updated_service = v1.read_namespaced_service(name = svc["metadata"]["name"], namespace="avi-system")
+    except ApiException as e:
+        print(e)
+
+def get_svc(config_file):
```

Comment:
get_svc name is misleading, it does get of services as well as update of annotation. I think a better naming is required here.

Reply:
Updated function name